### DISCUSSION TITLE
Anyone have a recommendation for an ad free Android TV / Fire Stick app?

### DISCUSSION THREAD

**Discussion by Jaladhjin** [bttv-android/bttv#549] [Q&A]
I see this part: bttv-android will not work on Android TVs as it is a mod of the Twitch mobile App and not of the Twitch Andorid TV App. You can install it, just do not expect a good experience

Which is okay obv this specific app won't work np np but thank you for it regardless.

Anyone know of any modded Twitch app for a Fire Stick that's > ads ? 😁

**Comment by NumbPat** [comment]
If the BTTV-Author is fine with advertising other projects, I could name one I discovered lately and loving it.

[S0undTV](https://github.com/S0und/S0undTV)
